### ISSUE TITLE
Login just working for the main user (not ...ftpX)

### ISSUE THREAD

**Issue by asdfklgash** [mmunz/fdcs#3]
It looks like the user is always just the main user (for example `main`) and not `mainftp1` during login (in `fdcs-entry`).
The container is created correctly as `fdcs_mainftp1`.

**Comment by mmunz** [comment]
Yes, afair there are no individual users from froxlor available besides the main user. I think it never worked with the additional users.
